README.md

```diff
@@ -58,32 +58,33 @@ const ethNetwork = 'ropsten'; // or mainnet;
 // Use the helper function to get the config
 const config = getConfig(ethNetwork);
 
-// Setup a provider and signer
+// Setup a provider and a signer
 const privateKey = YOUR_PRIVATE_KEY;
 const provider = new AlchemyProvider(ethNetwork, YOUR_ALCHEMY_API_KEY);
 const signer = new Wallet(privateKey).connect(provider);
 ```
 
-#### Stark Wallet
+#### WalletConnection
 
-Some methods require a stark wallet as a parameter. The Core SDK expects you will generate your own stark wallet.
+WalletConnection is a top level connector which contains all the needed resources (eg.: signers for L1 and L2) to possibilitate Core SDK workflows usage.
```

Comment:
@sprngbk can you please help polish the wording here 🙏

Reply:
> "WalletConnection is a high-level connector that contains all necessary resources (eg.: signers for L1 and L2) that make it possible to operate through the Core SDK workflows."

@Kastet  Wdyt?